<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>395</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>482</width>
    <height>395</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>482</width>
    <height>395</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>482</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EduTest</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Logo.qrc">
    <normaloff>:/logo/WindowLogo.png</normaloff>:/logo/WindowLogo.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="WelcomeMessage">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка авторизации.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartTask1Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartTask2Button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartTask3Button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartTask4Button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartTask5Button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 5</string>
    </property>
   </widget>
   <widget class="QTableView" name="StatView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>441</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="StatLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePasswordButton">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>340</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сменить пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Если задание выполнено правильно, вы зарабатываете один балл, но если его решение неверно, то вы теряете один балл.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Task1Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задача 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task2Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задача 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task3Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задача 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task4Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задача 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task5Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задача 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task1StatLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка при загрузке данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task2StatLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка при загрузке данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task3StatLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка при загрузке данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task4StatLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка при загрузке данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task5StatLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка при загрузке данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="SearchLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Начните вводить текст</string>
    </property>
   </widget>
   <zorder>WelcomeMessage</zorder>
   <zorder>StartTask1Button</zorder>
   <zorder>StartTask2Button</zorder>
   <zorder>StartTask3Button</zorder>
   <zorder>StartTask4Button</zorder>
   <zorder>StartTask5Button</zorder>
   <zorder>StatView</zorder>
   <zorder>StatLabel</zorder>
   <zorder>ExitButton</zorder>
   <zorder>ChangePasswordButton</zorder>
   <zorder>InfoLabel</zorder>
   <zorder>Task3StatLabel</zorder>
   <zorder>Task5Label</zorder>
   <zorder>Task3Label</zorder>
   <zorder>Task5StatLabel</zorder>
   <zorder>Task4Label</zorder>
   <zorder>Task2StatLabel</zorder>
   <zorder>Task1Label</zorder>
   <zorder>Task1StatLabel</zorder>
   <zorder>Task4StatLabel</zorder>
   <zorder>Task2Label</zorder>
   <zorder>SearchLabel</zorder>
   <zorder>SearchLine</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>StartTask1Button</tabstop>
  <tabstop>StartTask2Button</tabstop>
  <tabstop>StartTask3Button</tabstop>
  <tabstop>StartTask4Button</tabstop>
  <tabstop>StartTask5Button</tabstop>
  <tabstop>StatView</tabstop>
  <tabstop>SearchLine</tabstop>
  <tabstop>ChangePasswordButton</tabstop>
  <tabstop>ExitButton</tabstop>
 </tabstops>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
